ibis/impala/tests/test_window.py

```diff
@@ -129,6 +123,15 @@ def test_window_frame_specs(con, window, frame):
     assert_sql_equal(expr, expected)
 
 
+def test_window_rows_with_max_lookback(con):
+    with pytest.raises(NotImplementedError):
```

Comment:
Can you put all of the code that doesn't raise an exception outside the `with` block? That way if something else raises that error spuriously we don't swallow it and think that that code is working.

Reply:
Yes, fixed in next commit